tests/Functional/BigIntUnsignedConversionTest.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL\Tests\Functional;
+
+use Doctrine\DBAL\Platforms\MySQLPlatform;
+use Doctrine\DBAL\Tests\FunctionalTestCase;
+use Doctrine\DBAL\Types\BigIntType;
+
+use function sprintf;
+
+use const PHP_INT_MAX;
+
+class BigIntUnsignedConversionTest extends FunctionalTestCase
+{
+    private BigIntType $typeInstance;
+
+    protected function setUp(): void
+    {
+        if (
+            ! $this->connection->getDatabasePlatform() instanceof MySQLPlatform
+        ) {
+            self::markTestSkipped('This test only works on MySQL');
+        }
+
+        $this->typeInstance = new BigIntType();
+
+        $this->connection->executeStatement(
+            <<<'SQL'
+CREATE TABLE IF NOT EXISTS doctrine_tests.mysql_unsigned_test (
```

Comment:
This should use the schema API so that it does not hardcode support for mysql only